src/components/Coaching/CoachingDetail/Activity/Activity.test.tsx

```diff
@@ -236,21 +249,21 @@ describe('Activity', () => {
     const { getByTestId } = render(<TestComponent />);
 
     await waitFor(() =>
```

Comment:
I think we should be checking that the totals are correct in each of the boxes, not just the header text.